spec/fixtures/external_pager.rb

```diff
@@ -0,0 +1,8 @@
+output_file = ARGV.first or raise "No output file given!"
+
+STDIN.each_line do |line|
+  File.open(output_file, 'a') do |output|
+    output.puts(line)
+    output.flush
+  end
+end
```

Comment:
This works for me, e.i. pies to the output file for the two integration tests when I replace it with:

```ruby
STDOUT.puts(STDIN.each_line)
```